requirements.txt

```diff
@@ -43,7 +43,7 @@ botocore>=1.12.13
 cbor2>=5.2.0
 crontab>=0.22.6
 cryptography
-elasticsearch>=7.0.0,<7.14.0
+opensearch-py>=1.0.0,<2.0.0
```

Comment:
Is there a reason for the dependency boundaries? We usually don't limit the dependencies if it's not necessary.